### PR TITLE
fix(router) correct assignments from ngx.re.sub

### PR DESCRIPTION
**Pull request by p0pr0ck5**

### CODE REVIEW

File: kong/core/router.lua [hunk 1]
```diff
@@ -650,7 +650,7 @@ function _M.new(apis)
 
     else
       local err
-      new_uri, err = re_sub(uri, api_t.strip_uri_regex, "/$1", "ajo")
+      new_uri, _, err = re_sub(uri, api_t.strip_uri_regex, "/$1", "ajo")
```
Comment (thibaultcha) [inline kong/core/router.lua]: :grimacing:

However... that's now a global I think :wink:

Comment (p0pr0ck5) [inline kong/core/router.lua]: BRAAP. thanks.

**Comment by bungle** [comment]
Good catch!

File: kong/core/router.lua (hunk 1 above)
Comment (bungle) [inline kong/core/router.lua]: @thibaultcha damn, you have great eyes!

**Pull request merged**